### ISSUE TITLE
Missing  Development Dependencies for Debian/Ubuntu/Mint

### ISSUE THREAD

**Issue by fanaticalfishing** [firoorg/firo#1149]
autoconf 
make 
gcc 
g++

**Comment by justanwar** [comment]
Which specific versions of the distribution?

I did the following on Ubuntu 18.04, per the readme:

`sudo apt-get install git curl python build-essential libtool automake pkg-config cmake`

Checking it afterwards:

```
justanwar@ubuntu1804:~$ sudo apt-get install autoconf make gcc g++
Reading package lists... Done
Building dependency tree
Reading state information... Done
autoconf is already the newest version (2.69-11).
autoconf set to manually installed.
make is already the newest version (4.1-9.1ubuntu1).
make set to manually installed.
g++ is already the newest version (4:7.4.0-1ubuntu2.3).
g++ set to manually installed.
gcc is already the newest version (4:7.4.0-1ubuntu2.3).
gcc set to manually installed.
0 upgraded, 0 newly installed, 0 to remove and 20 not upgraded.
```

The build system and documentation were updated with #1138

**Comment by fanaticalfishing** [comment]
20.04 ubuntu

I tried to build it as per the instructions on a new vps. It would not because autoconf make gcc g++ were not installed. The list of dependencies for redhat includes them, but not ubuntu.

**Comment by fanaticalfishing** [comment]
Digital Ocean.
let me try again

**Comment by fanaticalfishing** [comment]
> 

It works.
I'm not sure why it didn't work twice before. But it works now
Thank you for your diligence
